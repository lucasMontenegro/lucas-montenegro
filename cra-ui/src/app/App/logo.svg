<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->
<svg xmlns="http://www.w3.org/2000/svg" height="512" width="512" version="1.1" viewBox="0 0 135.46666 135.46667">
 <path d="m67.691 24.236s-42.339 0.000038-52.965 2.5339c-5.6898 1.459-10.375 5.758-11.965 11.056-2.7609 9.751-2.761 29.946-2.761 29.946s0.00008 20.271 2.7613 29.869c1.5898 5.298 6.1912 9.5205 11.965 10.979 10.71 2.6106 52.965 2.6107 52.965 2.6107s42.423-0.00004 53.049-2.5339c5.7735-1.4589 10.375-5.6047 11.881-10.979 2.8449-9.6747 2.845-29.869 2.845-29.869s0.0835-20.271-2.845-30.023c-1.5061-5.298-6.1075-9.5205-11.881-10.903-10.626-2.6874-53.049-2.6875-53.049-2.6875z" fill="#fff"/>
 <g transform="matrix(.86095 0 0 .86095 9.4339 9.4186)">
  <path d="m28.907 83.848h23.915v10.294h-37.535z" fill="#fbbc05"/>
  <path d="m15.287 41.324h13.62v42.523l-13.62 10.295z" fill="#4285f4"/>
  <path d="m107.23 41.324c2.9397 0.000038 12.913 0.000094 12.913 0.000094l-0.00009 52.818h-12.913z" fill="#4285f4"/>
  <path d="m61.419 41.324c2.9397 0.000038 12.913 0.000094 12.913 0.000094l-0.0001 52.818h-12.913z" fill="#34a853"/>
  <path d="m74.331 41.324 4.422-0.000094 12.028 28.266 12.099-28.266 4.4222 0.000094v14.187l-12.167 28.478h-8.632l-12.17-28.478c-0.000099 0 0-14.186 0-14.186z" fill="#ea4335"/>
 </g>
</svg>
