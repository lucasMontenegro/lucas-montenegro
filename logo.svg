<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   height="512"
   width="512"
   version="1.1"
   viewBox="0 0 135.46666 135.46667"
   id="svg4567"
   sodipodi:docname="logo.svg"
   inkscape:version="0.92.1 r15371">
  <metadata
     id="metadata4573">
    <rdf:RDF>
      <cc:Work
         rdf:about="">
        <dc:format>image/svg+xml</dc:format>
        <dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
        <dc:title />
      </cc:Work>
    </rdf:RDF>
  </metadata>
  <defs
     id="defs4571" />
  <sodipodi:namedview
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1"
     objecttolerance="10"
     gridtolerance="10"
     guidetolerance="10"
     inkscape:pageopacity="0"
     inkscape:pageshadow="2"
     inkscape:window-width="1362"
     inkscape:window-height="725"
     id="namedview4569"
     showgrid="false"
     inkscape:zoom="0.48632813"
     inkscape:cx="352.52486"
     inkscape:cy="215.31546"
     inkscape:window-x="0"
     inkscape:window-y="20"
     inkscape:window-maximized="0"
     inkscape:current-layer="svg4567"
     inkscape:snap-bbox="true"
     inkscape:snap-page="true" />
  <rect
     style="fill:#02838c;fill-rule:evenodd;stroke-width:0;fill-opacity:1"
     id="rect4549"
     x="2.7271001e-06"
     y="2.7271001e-06"
     width="135.47"
     height="135.47"
     ry="1.0981e-13"
     rx="35.563999" />
  <path
     d="m 61.11868,70.202639 a 19.843599,10.847834 0 0 0 -19.8436,10.847834 v 10.318672 a 2.8222008,2.8222008 0 0 0 2.822201,2.8222 h 47.271864 a 2.8222008,2.8222008 0 0 0 2.8222,-2.8222 V 81.050473 A 19.843599,10.847834 0 0 0 74.347746,70.202639 h -6.614533 z"
     id="path4551"
     inkscape:connector-curvature="0"
     style="fill:#ffffff;fill-rule:evenodd;stroke-width:0.25779042" />
  <path
     d="M 80.433623,53.97549 A 12.700063,12.700063 0 0 1 67.733559,66.675554 12.700063,12.700063 0 0 1 55.033496,53.97549 12.700063,12.700063 0 0 1 67.733559,41.275427 12.700063,12.700063 0 0 1 80.433623,53.97549 Z"
     id="path4553"
     inkscape:connector-curvature="0"
     style="fill:#ffffff;fill-rule:evenodd;stroke-width:0.17106086" />
  <g
     aria-label="{"
     transform="matrix(0.9886671,0,0,0.95685587,1.9973714,0.0981112)"
     style="font-size:110.8628006px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;stroke-width:1.00002527"
     id="text4559">
    <path
       d="m 46.685881,114.04477 v 7.79504 h -3.356198 q -13.478925,0 -18.080164,-4.00578 -4.547107,-4.00578 -4.547107,-15.96901 V 88.927421 q 0,-8.173966 -2.92314,-11.313635 -2.92314,-3.13967 -10.6099166,-3.13967 H 3.8672895 v -7.740908 h 3.3020659 q 7.7409086,0 10.6099166,-3.085537 2.92314,-3.139669 2.92314,-11.205371 V 39.450566 q 0,-11.963222 4.547107,-15.914875 4.601239,-4.005785 18.080164,-4.005785 h 3.356198 v 7.740909 H 43.00489 q -7.632644,0 -9.96033,2.381818 -2.327686,2.381818 -2.327686,10.014462 v 13.424792 q 0,8.498759 -2.490082,12.342148 -2.435951,3.843388 -8.390496,5.196693 6.008678,1.46157 8.444628,5.304959 2.43595,3.843388 2.43595,12.288015 v 13.424788 q 0,7.63265 2.327686,10.01447 2.327686,2.38181 9.96033,2.38181 z"
       style="fill:#ffffff;stroke-width:0.83147103"
       id="path4599"
       inkscape:connector-curvature="0" />
  </g>
  <g
     aria-label="{"
     transform="matrix(-0.98866716,0,0,0.95685587,5.1257438,0.0981112)"
     style="font-size:110.8628006px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;stroke-width:1.00002527"
     id="text4563">
    <path
       d="m -83.128841,114.04477 v 7.79504 h -3.356198 q -13.478925,0 -18.080161,-4.00578 -4.54711,-4.00578 -4.54711,-15.96901 V 88.927421 q 0,-8.173966 -2.92314,-11.313635 -2.92314,-3.13967 -10.60992,-3.13967 h -3.30206 v -7.740908 h 3.30206 q 7.74091,0 10.60992,-3.085537 2.92314,-3.139669 2.92314,-11.205371 V 39.450566 q 0,-11.963222 4.54711,-15.914875 4.601236,-4.005785 18.080161,-4.005785 h 3.356198 v 7.740909 h -3.680992 q -7.632643,0 -9.960329,2.381818 -2.327686,2.381818 -2.327686,10.014462 v 13.424792 q 0,8.498759 -2.490082,12.342148 -2.43595,3.843388 -8.3905,5.196693 6.00868,1.46157 8.44463,5.304959 2.435952,3.843388 2.435952,12.288015 v 13.424788 q 0,7.63265 2.327686,10.01447 2.327686,2.38181 9.960329,2.38181 z"
       style="fill:#ffffff;stroke-width:0.83147103"
       id="path4602"
       inkscape:connector-curvature="0" />
  </g>
</svg>
