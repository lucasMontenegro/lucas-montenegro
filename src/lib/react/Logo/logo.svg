<svg xmlns="http://www.w3.org/2000/svg" height="512" width="512" version="1.1" viewBox="0 0 135.46666 135.46667">
 <rect fill-rule="evenodd" rx="35.564" ry="1.0981e-13" height="135.47" width="135.47" y=".0000027271" x=".0000027271" stroke-width="0" fill="#00646b"/>
 <g fill="#fff">
  <path d="m61.119 70.203a19.844 10.848 0 0 0 -19.844 10.848v10.319a2.8222 2.8222 0 0 0 2.8222 2.8222h47.272a2.8222 2.8222 0 0 0 2.8222 -2.8222v-10.32a19.844 10.848 0 0 0 -19.843 -10.847h-6.6145z" fill-rule="evenodd" stroke-width=".25779"/>
  <path d="m80.434 53.975a12.7 12.7 0 0 1 -12.7 12.701 12.7 12.7 0 0 1 -12.701 -12.701 12.7 12.7 0 0 1 12.701 -12.7 12.7 12.7 0 0 1 12.7 12.7z" fill-rule="evenodd" stroke-width=".17106"/>
  <g aria-label="{" transform="matrix(.98867 0 0 .95686 1.9974 .098111)">
   <path stroke-width=".83147" fill="#fff" d="m46.686 114.04v7.795h-3.3562q-13.479 0-18.08-4.0058-4.5471-4.0058-4.5471-15.969v-12.933q0-8.174-2.9231-11.314-2.9231-3.1397-10.61-3.1397h-3.3027v-7.7409h3.3021q7.7409 0 10.61-3.0855 2.9231-3.1397 2.9231-11.205v-12.991q0-11.963 4.5471-15.915 4.6012-4.0058 18.08-4.0058h3.3562v7.7409h-3.68q-7.6326 0-9.9603 2.3818t-2.3277 10.014v13.425q0 8.4988-2.4901 12.342-2.436 3.8434-8.3905 5.1967 6.0087 1.4616 8.4446 5.305 2.436 3.8434 2.436 12.288v13.425q0 7.6326 2.3277 10.014 2.3277 2.3818 9.9603 2.3818z"/>
  </g>
  <g aria-label="{" transform="matrix(-.98867 0 0 .95686 5.1257 .098111)">
   <path stroke-width=".83147" fill="#fff" d="m-83.129 114.04v7.795h-3.3562q-13.479 0-18.08-4.0058-4.5471-4.0058-4.5471-15.969v-12.933q0-8.174-2.9231-11.314-2.9231-3.1397-10.61-3.1397h-3.3021v-7.7409h3.3021q7.7409 0 10.61-3.0855 2.9231-3.1397 2.9231-11.205v-12.991q0-11.963 4.5471-15.915 4.6012-4.0058 18.08-4.0058h3.3562v7.7409h-3.681q-7.6326 0-9.9603 2.3818t-2.3277 10.014v13.425q0 8.4988-2.4901 12.342-2.436 3.8434-8.3905 5.1967 6.0087 1.4616 8.4446 5.305 2.436 3.8434 2.436 12.288v13.425q0 7.6326 2.3277 10.014 2.3277 2.3818 9.9603 2.3818z"/>
  </g>
 </g>
</svg>
