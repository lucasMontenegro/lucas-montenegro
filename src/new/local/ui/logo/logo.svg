<svg xmlns="http://www.w3.org/2000/svg" height="512" width="512" version="1.1" viewBox="0 0 135.46666 135.46667">
 <g>
  <rect rx="10.314" ry="1.206e-13" height="135.47" width="135.47" y=".0000027271" x=".0000027271" stroke-width="0" fill="#fff"/>
  <rect rx="10454" ry="7.1632e-14" height="16.933" width="101.6" y="101.6" x="16.933" stroke-width="0" fill="#34a853"/>
  <g aria-label="L" transform="matrix(2.9629 0 0 2.9629 -22.483 -180.9)" fill="#4285f4">
   <path d="m13.303 66.77h6.6318v20.705h11.644v5.0126h-18.276z" stroke-width=".26458" fill="#4285f4"/>
  </g>
  <path d="m65.923 16.933h15.52l10.769 25.308 10.828-25.308h15.489v47.289h-11.53v-34.587l-10.892 25.497h-7.728l-10.896-25.497v34.587h-11.561z" fill="#ea4335"/>
  <rect rx="4010.9" ry="8.6484e-14" height="20.444" width="38.982" y="72.689" x="79.552" stroke-width="0" fill="#fbbc05"/>
 </g>
</svg>
