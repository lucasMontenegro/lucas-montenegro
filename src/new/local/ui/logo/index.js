import React from "react"
import { ReactComponent as Logo } from "./logo.svg"
export default (<Logo width="32" height="32" aria-label="Lucas Montenegro logo" />)