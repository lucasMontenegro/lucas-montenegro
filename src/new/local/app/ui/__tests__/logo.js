import renderer from "react-test-renderer"
import logo from "new/local/app/ui/logo"
describe(`new/local/app/ui/logo`, () => {
  it(`should render`, () => {
    expect(renderer.create(logo).toJSON()).toMatchSnapshot()
  })
})