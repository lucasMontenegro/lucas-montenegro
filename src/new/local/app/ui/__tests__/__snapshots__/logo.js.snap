// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new/local/app/ui/logo should render 1`] = `
<svg
  aria-label="Lucas Montenegro logo"
  height="32"
  width="32"
>
  logo.svg
</svg>
`;
